README.md

```diff
@@ -53,31 +53,53 @@ Install NPM package
     import {HDKEY, DID} from '@functionland/fula-sec'
 
 
-    // Prefix moc keys
-    let secretKey = '123456789'
-    let signedKey = '9d7020006cf0696334ead54fffb859a8253e5a44860c211d23c7b6bf842d0c63535a5efd266a647cabdc4392df9a4ce28db7dc393318068d93bf33a32adb81ae';
+    /* Prefix moc keys */
+    let password = '123456789'  //User`s password
+    let signedKey = '9d7020006cf0696334ead54fffb859a8253e5a44860c211d23c7b6bf842d0c63535a5efd266a647cabdc4392df9a4ce28db7dc393318068d93bf33a32adb81ae'; // signedKey from metamask
```

Comment:
can you add a comment:
// signedKey is the signature coming from the locally-running service of a 3rd party signing authority like Meta Mask Wallet by signing part of the password (not the full password is being sent to the 3rd party signing authority)